config/toggles.js

```diff
@@ -126,6 +126,8 @@ const shortTermToggles = {
 
   returnUser: environmentVariableTrue(pe, 'RETURN_CREATEDBY_ON_TOKEN_INPUT'),
 
+  checkCacheOnPost: environmentVariableTrue(pe, 'CHECK_CACHE_ON_POST'),
+
```

Comment:
Please use "FAST_FAIL_DUPLICATE_SUBJECT" as the environment variable name, and "fastFailDuplicateSubject" as the toggle name.